client/src/partials/journal/modals/journalErrorViewer.modal.js

```diff
@@ -0,0 +1,46 @@
+angular.module('bhima.controllers')
+  .controller('JournalErrorViewerModalController', JournalErrorViewerModalController);
+
+JournalErrorViewerModalController.$inject = [
+  '$uibModalInstance', 'GridColumnService', 'GridGroupingService', 'JournalErrorViewerService', 'params'
+];
+
+/**
+ * @module journal/modals/journalErrorViewer.modal
+ *
+ * @description
+ * This controller provides an error report relative to the posting operation
+ */
+function JournalErrorViewerModalController (ModalInstance, Columns, Grouping, JournalErrorViewerService, Params) {
+  var vm = this;
+  var cacheKey = 'errorReport';
+  var columns = [
+    { field : 'code', displayName : 'TABLE.COLUMNS.ERROR_TYPE', headerCellFilter : 'translate', enableCellEdit : false, allowCellFocus : false},
+    {field : 'transaction', displayName : 'TABLE.COLUMNS.TRANSACTION', headerCellFilter : 'translate', enableCellEdit : false, allowCellFocus : false}
```

Comment:
Actually, since this is for ErrorViewerModal, do you even need `enableCellEdit`?  You didn't but the `ui-grid-edit` directive on the grid...  How is editing enabled?